### PR TITLE
allow unset TSD_GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by tbg**
otherwise, when you don't set it in your fork,

> - build/builder.sh build/circle-deps.sh docker
> 
> build/circle-deps.sh returned exit code 1
> 
> build/builder.sh: line 52: TSD_GITHUB_TOKEN: unbound variable Action failed: build/circle-deps.sh

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,24 +50,41 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
...
-`git commit -a -m 'update CONTRIBUTING.md'`
-
-+ Run tests. It's usually enough to run `make test testrace`. You can also run `make acceptance` to have better test coverage. Running acceptance tests requires the Docker setup.
+  `git checkout -b $USER/update-readme`
+
++ Hack away and commit your changes locally using `git add` and `git commit`. Remember to write tests! The following are helpful for running specific subsets of tests:
+  ```
+  make test`
+  # Run all tests in ./storage
+  make test PGK=./storage
```
Comment (petermattis) [inline CONTRIBUTING.md]: `s/PGK/PKG/g`

Comment (tschottdorf) [inline CONTRIBUTING.md]: done.

**Comment by tschottdorf** [comment]
updated, PTAL

**Comment by tamird** [comment]
few more nits. LGTM

**Comment by tschottdorf** [comment]
both addressed.

**Pull request merged**